ts/util/logger/buildPinoLogger.ts

```diff
@@ -0,0 +1,34 @@
+import pino, { type StreamEntry } from 'pino';
+import { createRotatingPinoDest } from './rotatingPinoDest';
+
+export function buildPinoLogger(logFile: string, onStreamClosed: () => void) {
+  const hasDebugLoggingEnabled = !!process.env.SESSION_DEBUG;
+  const rotatingStream = createRotatingPinoDest({
+    logFile,
+  });
+
+  rotatingStream.on('close', onStreamClosed);
+  rotatingStream.on('error', onStreamClosed);
+
+  const streams = new Array<StreamEntry>();
+
+  const levelToLog: pino.Level = hasDebugLoggingEnabled ? 'debug' : 'info';
+
+  streams.push({ level: levelToLog, stream: rotatingStream });
+  streams.push({
+    level: levelToLog,
+    stream: process.stdout,
```

Comment:
Maybe we only want sdout when in debug mode?

Reply:
I don't think so. It's a pain to set on windows to set an env and having those on will help us debug a node issue faster.